packages/fetchai/skills/aries_faber/handlers.py

```diff
@@ -101,11 +98,28 @@ def _send_default_message(self, content: Dict) -> None:
         # send
         self.context.outbox.put_message(message=message)
 
+    def _register_public_did(self) -> None:
+        """Register DID on the ledger."""
+        strategy = cast(Strategy, self.context.strategy)
+        self.context.behaviours.faber.send_http_request_message(
+            method="POST",
+            url=strategy.admin_url
+            + ADMIN_COMMAND_REGISTGER_PUBLIC_DID
+            + f"?did={self.did}",
+            content="",
+        )
+
```

Comment:
What is the diff between this and "_register_did" below?

Reply:
updates